radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/controller/HealthController.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * (C) Copyright 2021 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.api.controller;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.radixdlt.api.Controller;
+import com.radixdlt.api.service.NetworkInfoService;
+
+import io.undertow.server.HttpServerExchange;
+import io.undertow.server.RoutingHandler;
+
+import static com.radixdlt.api.JsonRpcUtil.jsonObject;
+import static com.radixdlt.api.RestUtils.respond;
+
+public class HealthController implements Controller {
+	private final NetworkInfoService networkInfoService;
+
+	public HealthController(NetworkInfoService networkInfoService) {
+		this.networkInfoService = networkInfoService;
+	}
+
+	@Override
+	public String root() {
+		return "/health";
+	}
+
+	@Override
+	public void configureRoutes(final RoutingHandler handler) {
+		handler.get("/health", this::handleHealthRequest);
+		handler.get("/health/", this::handleHealthRequest);
```

Comment:
Minor: don't think we need both; it only causes confusion rather than does anything good.

Reply:
It makes both paths accepted by server. Such a minor thing as trailing slash should not cause issues to client

Reply:
I understand your intention but not totally agree. 

We should encourage clients to use the **correct** endpoint (less flexible), rather than to correct their mistakes.